protein/api-src/org/labkey/api/protein/ProteomicsModule.java

```diff
@@ -1,29 +1,25 @@
-/*
- * Copyright (c) 2007-2008 LabKey Corporation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+/*
+ * Copyright (c) 2013-2016 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.labkey.api.protein;
 
-package org.labkey.ms2.protein;
-
-/**
- * User: jeckels
- * Date: Apr 3, 2007
- */
-public class ProteinException extends Exception
-{
-    public ProteinException(String s)
-    {
-        super(s);
-    }
-}
+import org.labkey.api.module.Module;
+
+/**
+ * Tagging interface for module that need to make sure that the protein table is available.
+ */
+public interface ProteomicsModule extends Module
```

Comment:
Seems like a strange sequence in the refactor - moving a file and then repurposing it for a different class. Fine, just strange.

Reply:
Not my doing... Git is just confusing a deleted class with a moved class